src/components/About.js

```diff
@@ -4,7 +4,7 @@ export default function aboutPage() {
   return (
     <div>
       <h1>About Us!</h1>
-      <h3>Learn more about our team and our business and all you could ever know ever!!</h3>
+      <p>Learn more about our team and our business and all you could ever know ever!!</p>
```

Comment:
I'd like to see something about either 
-what your product does OR
-your dev team.